src/framework/theme/components/calendar/base-calendar.component.ts

```diff
@@ -72,9 +72,11 @@ export class NbBaseCalendarComponent<D, T> implements OnInit {
   @Input() visibleDate: D;
 
   /**
-   * Determines should we show calendars header or not.
+   * Determines should we show calendars navigation or not.
```

Comment:
```suggestion
   * Determines whether we should show calendar navigation or not.
```
I believe it will sound more native